mypy/test/teststubgen.py

```diff
@@ -110,7 +125,7 @@ def test_module_not_found(self) -> None:
                 os.chdir(tmp)
                 self.make_file(tmp, "mymodule.py", content="import a")
                 opts = parse_options(["-m", "mymodule"])
-                py_mods, c_mods = collect_build_targets(opts, mypy_options(opts))
+                py_mods, c_mods, ignored_py_mods = collect_build_targets(opts, mypy_options(opts))
```

Comment:
`ignored_py_mods` var is not asserted here.
Do we have something to check here?

Reply:
Well I checked this test case and found its name `test_module_not_found` is confusing because module is found at this case.

I confirmed that I didn't make any unintentional changes by checking out master branch of mypy, and modified `mypy/test/teststubgen.py` around L103 like below and run `pytest -k StubgenCmdLine -vvv`.
The result of the `assert_equal(c_mods, [])` is fine but `assert_equal(py_mods, [])` reports error because mymodule was found.
```
    @unittest.skipIf(sys.platform == "win32", "clean up fails on Windows")
    def test_module_not_found(self) -> None:
        current = os.getcwd()
        captured_output = io.StringIO()
        sys.stdout = captured_output
        with tempfile.TemporaryDirectory() as tmp:
            try:
                os.chdir(tmp)
                self.make_file(tmp, "mymodule.py", content="import a")
                opts = parse_options(["-m", "mymodule"])
                py_mods, c_mods = collect_build_targets(opts, mypy_options(opts))
                assert captured_output.getvalue() == ""
                assert_equal(c_mods, [])
>               assert_equal(py_mods, [])
E               AssertionError: [<mypy.stubgen.StubSource object at 0x113393190>] != []
```

I fixed test case name and add assertions for variables py_mods, c_mods, ignored_py_mods by [4f44401](https://github.com/python/mypy/pull/13919/commits/4f44401ba9491070911ed173214b130e50df4a29).